.github/workflows/ui-tests.yml

```
@@ -0,0 +1,96 @@
+          
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+          
+      - name: Setup pnpm config
+        run: pnpm config set store-dir ~/.pnpm-store
+      
+      # Add caching for faster builds
+      - name: Cache pnpm modules
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)